.eslintrc

```diff
@@ -147,7 +147,14 @@
       "error",
       "consistent"
     ],
-    "quotes": "error",
+    "quotes": [
```

Comment:
Let's not change the style rules here. Either change the code to pass, or if it cannot be changed and there is a good reason to make an exception, use an inline comment to make an exception to the relevant code line(s). See https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments